docs/_documentations/mdt-che-installinfo.md

```diff
@@ -139,12 +139,12 @@ Go to **View**>**Find Command…**>**Codewind: Build**.
 ## Updating the version
 Restart the Codewind workspace in Che. Che automatically pulls the newest version of Codewind and the Theia extension.
 
-# Installing Codewind on OpenShift
+## Installing Codewind on OpenShift
```

Comment:
We don't need this heading, `installing Codewind on OpenShift` uses the same way as `Creating the Codewind workspace with a Devfile`, so it's duplicated heading.

Reply:
Removed